recipe/meta.yaml

```diff
@@ -188,7 +187,7 @@ outputs:
       run:
         - python
         - {{ pin_compatible('fastrlock', max_pin='x.x') }}
-        - numpy ~=1.22
+        - numpy ~=1.22  # TODO: actually requires NumPy>=2 probably.
```

Comment:
```suggestion
        - numpy >=1.22,<3.0.0a0  # TODO: actually requires NumPy>=2 probably.
```

Reply:
Might as well put this in also then, I think, thanks.